src/zenml/model/__init__.py

```diff
@@ -12,23 +12,4 @@
 #  or implied. See the License for the specific language governing
 #  permissions and limitations under the License.
 
-"""Initialization of ZenML model.
-ZenML model support Model Control Plane feature.
-"""
-
-from zenml.model.model_config import ModelConfig
```

Comment:
I'm not a fan of `ModelConfig` being removed from here and not placed under root zenml. Does this mean that people need to import like `from zenml.model.model_config import ModelCofig`? I assume that it is quite repeatable code.